modules/porous_flow/test/tests/jacobian/esbc01.i

```diff
@@ -0,0 +1,99 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+  at_nodes = true
+[]
+
+[UserObjects]
+  [./dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = 'pp temp'
+    number_fluid_phases = 1
+    number_fluid_components = 1
+  [../]
```

Comment:
Need to replace [./blah] ... [../] with [blah] ... []

Reply:
fixed